app/Http/Controllers/CommentsController.php

```diff
@@ -1,99 +0,0 @@
-<?php
-
-namespace App\Http\Controllers;
-
-use Illuminate\Http\Request;
-use App\Post;
-use App\Comment;
-
-class CommentsController extends Controller
-{
-    /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
-    {
-        //
-    }
-
-    /**
-     * Show the form for creating a new resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function create()
-    {
-        //
-    }
-
-    /**
-     * Store a newly created resource in storage.
-     *
-     * @param  \Illuminate\Http\Request  $request
-     * @return \Illuminate\Http\Response
-     */
-    public function store(Request $request)
-    {
-        //validate data
-        $attributes = $request->validate([
-            'name' => 'required',
-            'email' => 'required',
-            'comment' => 'required',
-        ]);
-        $attributes['post_id'] = $request->post_id;
```

Comment:
post_id should be put inside validate() as well. Plus check if the post actually exists.